build_and_push.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
```

Comment:
Maybe remove this and the mention to it from the Readme?

(just an idea... up to you to decide mate)

Reply:
I am going to leave it.  I modified it so it only works with this repo.